tests/helpers/compute_helpers.py

```diff
@@ -146,6 +146,19 @@ def get_compute_job_info(client, endpoint, params):
     return dict(job_info[0])
 
 
+def get_compute_result(client, endpoint, params):
+    response = client.get(
```

Comment:
can you use PreparedRequest here, too? (especially since params is already a dict)

Reply:
not possible, will fail because we don't have the full url (schema, host) in the tests